forms/FormField.php

```diff
@@ -1260,4 +1260,24 @@ public function castedCopy($classOrCopy) {
 		return $field;
 	}
 
+	/**
+	 * Determine if escaping of this field should be disabled
+	 *
+	 * @param bool $dontEscape
+	 * @return $this
+	 */
+	public function setDontEscape($dontEscape) {
```

Comment:
I dislike having negated method names, but this was added to provide a setter for an existing property.